.github/dependabot.yml

```diff
@@ -12,4 +12,5 @@ updates:
       # pin Jetty dependencies to 9.4.x
       - dependency-name: "org.eclipse.jetty"
         versions: "[9.5,)"
-      - dependency-name: "org.apache.hadoop"
\ No newline at end of file
+      - dependency-name: "org.apache.hadoop"
+      - dependency-name: "org.apache.calcite"
```

Comment:
```suggestion
      # We cannot upgrade calcite without shading their guava dependencies. 
      # We can't upgrade our own guava because we have to support Hadoop 2.
      # See https://github.com/apache/druid/pull/12258
      - dependency-name: "org.apache.calcite"
```